.github/workflows/run.yml

```diff
@@ -24,4 +28,4 @@ jobs:
           elixir-version: '1.14'
       - run: mix deps.get
       - run: mix release
-      - run: _build/prod/rel/roule_avec_spot/bin/roule_avec_spot start
+      - run: mix playlist
```

Comment:
Naive question: You might not need mix release anymore, or did I miss something?

Reply:
You're right. Fixed here: https://github.com/dany-kun/roule-avec-spot/pull/16